griptape/drivers/vector/base_vector_store_driver.py

```diff
@@ -72,6 +72,9 @@ def upsert_text(
             **kwargs
         )
 
+    def delete(self):
+        raise NotImplementedError("This Method is not implemented yet")
```

Comment:
Let's update the message to match other `NotImplementedError`s in the codebase. Something like:
`raise NotImplementedError(f"{self.__class__.__name__} does not support deletion.")`